.github/workflows/ee-build.yml

```diff
@@ -0,0 +1,215 @@
+# Combine workflow for pull-request, push-to-main and release events.
+
+name: EE Build
+
+on:
```

Comment:
To make this reusable, I think you'll need workflow_call here, otherwise it will run in this repo...